src/notifications.c

```diff
@@ -56,7 +58,13 @@ int HashNotificationCallback(RedisModuleCtx *ctx, int type, const char *event,
                     *hincrbyfloat_event = 0, *hdel_event = 0, *del_event = 0, *set_event = 0,
                     *rename_from_event = 0, *rename_to_event = 0, *trimmed_event = 0,
                     *restore_event = 0, *expired_event = 0, *evicted_event = 0, *change_event = 0,
-                    *loaded_event = 0;
+                    *loaded_event = 0, *json_set_event = 0;
+  
+  // TODO:clean
+  if (!strcmp("json_set", event) || !strcmp("json_incrby", event) || !strcmp("json_arrpop", event)) {

```

Comment:
Why the events uses '_' while the json commands uses '.'?

Reply:
What about change events that we get from crdt?

Reply:
The crdt will send a change event (not via json), we need to make sure we handle this event correctly.

Reply:
@MeirShpilraien 
> Why the events uses '_' while the json commands uses '.'?
@gkorland changed RedisJSON master to use `.` (also aligned with TimeSeries)

@ashtul We need to change it also in RediSearch